srcpkgs/bc/template

```diff
@@ -6,12 +6,13 @@ build_style=gnu-configure
 configure_args="--with-readline"
 hostmakedepends="ed flex"
 makedepends="readline-devel"
-short_desc="An arbitrary precision numeric processing language"
+short_desc="Arbitrary precision numeric processing language"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
-homepage="http://www.gnu.org/software/${pkgname}/"
 license="GPL-3"
```

Comment:
please use the spdx identifier for bc, it is a gnu project so it most likely is GPL-3.0-or-later but please check

Reply:
@maxice8 I just checked. You are correct.